docs/src/debug.md

```diff
@@ -6,49 +6,143 @@ title: "Debugging Tests"
+
+A browser window will open and the test will run and pause at where the breakpoint is set. You can step through the tests, pause the test and rerun the tests from the menu in VS Code.
+
+<img width="1269" alt="running test in debug mode" src="https://user-images.githubusercontent.com/13063165/212740233-3f278825-13e7-4a88-a118-dd4478d43a16.png" />
+
+### Debug in different Browsers
+
+By default debugging is done using the Chromium profile. You can debug your tests on different browsers by right clicking on the debug icon in the testing sidebar and clicking on the 'Select Default Profile' option from the dropdown.
+
+<img width="1312" alt="debugging on specific profile" src="https://user-images.githubusercontent.com/13063165/212879469-436f8130-c62a-49e1-9d67-c1903b478d5f.png" />
+
+Then choose the test profile you would like to use for debugging your tests. Each time you run your test in debug mode it will use the profile you selected. You can run tests in debug mode by right clicking the line number where your test is and selecting 'Debug Test' from the menu.
+
+<img width="1312" alt="choosing a profile for debugging" src="https://user-images.githubusercontent.com/13063165/212880198-eac22c3e-68ce-47da-9163-d6b376ae7575.png" />
+
+To learn more about debugging, see [Debugging in Visual Studio Code](https://code.visualstudio.com/docs/editor/debugging).
+
+
+## Playwright Inspector
 
-There are several ways of opening Playwright Inspector:
+The Playwright Inspector is a GUI tool to help you debug your Playwright tests. It allows you to step through your tests, live edit locators, pick locators and see actionability logs.
 
-#### --debug
+<img width="864" alt="Playwright Inspector" src="https://user-images.githubusercontent.com/13063165/212924587-4b84e5f6-b147-40e9-8c75-d7b9ab6b7ca1.png" />
+
+### Run in debug mode
 * langs: js
 
-* Debugging all Tests
+Run your tests with the `--debug` flag to open the inspector. This configures Playwright for debugging and opens the inspector. Additional useful defaults are configured when `--debug` is used:
 
-  ```bash
-  npx playwright test --debug
-  ```
+- Browsers launch in headed mode
+- Default timeout is set to 0 (= no timeout)
 
-* Debugging one test
+#### Debug all tests on all browsers
 
-  ```bash
-  npx playwright test example --debug
-  ```
+To debug all tests run the test command with the `--debug` flag. This will open the inspector and a browser window for each test and each browser. After stepping through the first test the inspector and browser window will close and the next test will run opening the inspector and a new browser window. Once all tests have finished running on the first browser configured in your [`playwright.config`](/test-configuration.md#multiple-browsers), the inspector and browser window will close and the next browser will open.
```

Comment:
you mean this `npx playwright test example.spec.ts:10 --debug` yes it should be explained in both. I am not used to command line since i use vscode extension so I am not used to adding in filters like test name. so yeh I think it should be in both cause if not people might not know that you can debug just one test